tests/test_dsp_pad_zeros.py

```diff
@@ -0,0 +1,48 @@
+import pyfar
+import numpy as np
+import pytest
+
+
+def test_zeropadding():
+    num_zeros = 100
+    n_samples = 1024
+
+    test_signal = pyfar.signals.impulse(
+        n_samples, delay=0, amplitude=np.ones((2, 3)), sampling_rate=44100)
```

Comment:
The testing might be easier to read if you use shorter signals, e.g.,
`num_zeros = 2`
`test_signal = pyfar.Signal([1, 0, 0, 0], 44100)`

Reply:
I'd like to also test n-dimensional signals here.